src/NewPkgEval.jl

```diff
@@ -19,20 +19,21 @@ builds_file() = joinpath(@__DIR__, "..", "deps", "Builds.toml")
 const skip_lists = Dict{String,Vector{String}}()
 
 """
-    get_registry()
+    get_registry([name]; update=false)
 
-Download the given registry, or if it already exists, update it. `name` must correspond
-to an existing stanza in the `deps/Registries.toml` file.
+Download the given registry, or if it already exists, update it if `update` is true.
+`name` must correspond to an existing stanza in the `deps/Registries.toml` file.
 """
-function get_registry(name=DEFAULT_REGISTRY)
+function get_registry(name=DEFAULT_REGISTRY; update=false)
```

Comment:
Is this really the right default? If you go in and want to do a PkgEval run, if you forget to manually update the registry you will run with old versions (perhaps very old if you run on a machine you only run PkgEval on).

Reply:
Maybe not, but `get_registry` isn't being called automatically at all so it's not particularly user friendly anyway. Maybe there should be an additional `run` function that takes package names, updates the registry, and runs PkgEval all together?

Reply:
It's called when you get the packages, no?

https://github.com/JuliaComputing/NewPkgEval.jl/blob/09244ce3d712617b80ee95083289cf79f9bbb670/src/NewPkgEval.jl#L117-L118

Reply:
I removed that because I wanted to get rid of the globally mutating behavior of what looks like a read-only function: https://github.com/JuliaComputing/NewPkgEval.jl/pull/28/files#diff-c7246d7c135b71eb409dff5b10be93d7R82
But you're right in that it hurts the simple use case. I'll try something.

Reply:
Okay makes sense. Another `run`  function that does the whole thing makes sense to me. 
```